docs/src/solutions.md

```diff
@@ -0,0 +1,105 @@
+```@meta
+CurrentModule = JuMP
+DocTestSetup = quote
+    using JuMP
+    const MOI = JuMP.MathOptInterface
+end
+```
+
+Querying Solutions
+==================
+
+So far we have seen all the elements and constructs related to writing a JuMP
+optimization model. In this section we reach the point of what to do
+with a solved problem. Suppose your model is named `model`. Right after the
+call to `JuMP.optimize!(model)` (which might take a while) it's possible to
+ask JuMP questions about the finished optimization step. Typical questions
+include: Why has the optimization process stopped? Do I have a solution to my
+problem? Is it optimal? Do I have a dual solution?
+
+JuMP follows closely the concepts defined in [MathOptInterface (MOI)](https://github.com/JuliaOpt/MathOptInterface.jl)
+to answer user questions about a finished call to `JuMP.optimize!(model)`.
+There are 3 main steps in querying a solution:
+
+* Firstly, we obtain the [`termination_status`](@ref) which will tell the user the reason why
+optimization stopped. An optimal solution could have been found, or the problem
```

Comment:
optimization stopped. *This could be due to a number of reasons:* an optimal solution could have been found, the problem ...